depdive/src/main.rs

```diff
@@ -0,0 +1,46 @@
+use anyhow::Result;
+use depdive::{
+    DependencyGraphAnalyzer, TabledCrateSourceDiffReport, TabledCratesioReport, TabledGitHubReport,
+};
+use guppy::MetadataCommand;
+use std::path::PathBuf;
+use tabled::{Style, Table};
+
+fn main() -> Result<()> {
+    let graph = MetadataCommand::new()
+        .current_dir(PathBuf::from("resources/test/valid_dep"))
+        .build_graph()?;
```

Comment:
Since this is the main binary you probably want this to run MetadatCommand from the current directory, not on some test fixture.

Reply:
done